NEWS.md

```diff
@@ -1,11 +1,12 @@
 # Release Notes
 
-## Version 3.2.4 (Release Candidate 1)
+## Version 3.2.4 (August 6, 2022)
 
 ### Corrections
 
  * Fixed a regression where shared folders could not be added using the Preferences dialog
  * Fixed a performance regression when updating file transfer lists
+ * Fixed a potential crash when downloading many large files at once
```

Comment:
` * Fixed a potential buffer overflow crash when displaying the estimated time of a very large download`

Reply:
buffer overflow is not relevant information for an average user